charts/hub-agent/templates/deployment.yaml

```diff
@@ -30,7 +30,9 @@ spec:
             - -add_dir_header
             - --enable-v1alpha1-apis={{ .Values.enableV1Alpha1APIs }}
             - --enable-v1beta1-apis={{ .Values.enableV1Beta1APIs }}
+            {{- if .Values.enableClusterInventoryAPI }}
```

Comment:
it depends on our binary. The current latest verified image does not have this flag.

If we pass this flag, the pod cannot be created because of this invalid flag.